### DISCUSSION TITLE
Better model for face embeddings?

### DISCUSSION THREAD

**Discussion by neckaros** [vladmandic/human#411] [General]
Hello,

I don't mind more complex model/less performance if it can improve face recognition. Do you know a good compatible model?

**Comment by vladmandic** [comment]
really good one is insightface. the problem with such models is not just performance, its the fact that they cannot run in browser, only on server-side.

**Reply by neckaros** [reply]
Thanks!
I run only on Node so not an issue!
InsightFace can be used with your Human library or I have to plug into it myself?

**Reply by neckaros** [reply]
Sorry just saw https://github.com/vladmandic/human/discussions/322 where you speak about this model!

Similarity seems to be always 0 except for an exact match. Do you know which config i should use to correctly scale similarity?
![image](https://github.com/vladmandic/human/assets/7789770/5edc3a50-6258-48b3-a0b5-f52ab54ba345)

Thanks again for this great library!
